Month.java

```diff
@@ -0,0 +1,49 @@
+public class Month {
+
+    public static void howDaysInMonth() {
```

Comment:
Please do not use specific values in the method body because your method should be universal and general. Add it as parameters
public static void daysOfTheMonth(int month) {
        switch (month) {
            case 1:
            case 3:
            case 5:
            case 7:
            case 8:
            case 10:
            case 12:
                System.out.println("31 days in this month");
                break;
            case 2:
                System.out.println("28 days in this month");
                break;
            case 4:
            case 6:
            case 9:
            case 11:
                System.out.println("30 days in this month");
                break;
            default:
                System.out.println("Incorrect month");
                break;
        }

To use the method in the main method:
daysOfTheMonth(3); // only here you can type month number as an argument